bin/gen-pgbouncer-conf.sh

```diff
@@ -89,7 +89,7 @@ EOFEOF
 rm -f /app/vendor/pgbouncer/users.txt
 if [ -n "$PGBOUNCER_STATS_USERNAME" ] && [ -n "$PGBOUNCER_STATS_PASSWORD" ]
 then
-    STATS_MD5_PASS="md5"`echo -n ${PGBOUNCER_STATS_USERNAME}${PGBOUNCER_STATS_PASSWORD} | md5sum | awk '{print $1}'`
+    STATS_MD5_PASS="md5"`echo -n ${PGBOUNCER_STATS_PASSWORD}${PGBOUNCER_STATS_USERNAME} | md5sum | awk '{print $1}'`
```

Comment:
Later in this file, in the upstream original, they compute:
```
DB_MD5_PASS="md5"`echo -n ${DB_PASS}${DB_USER} | md5sum | awk '{print $1}'`
```
...from which this line was based.  Notice how that original is password first, username second.